gtk/docs.md

```diff
@@ -1 +1,353 @@
+
+Also see `Widget::child_notify`.
+## `child`
+the child widget
+## `child_property`
+the name of a child property installed on
+ the class of `self`
+<!-- impl Expander::fn get_use_markup -->
+Returns whether the label’s text is interpreted as marked up with
+the [Pango text markup language](https://developer.gnome.org/pygtk/stable/pango-markup-language.html).
+See `Expander::set_use_markup`.
+
+# Returns
+
+`true` if the label’s text will be parsed for markup
+<!-- impl Expander::fn set_use_markup -->
+Sets whether the text of the label contains markup in
+[Pango’s text markup language](https://developer.gnome.org/pygtk/stable/pango-markup-language.html).
+See `Label::set_markup`.
+## `use_markup`
+`true` if the label’s text should be parsed for markup
+<!-- struct Label -->
+The `Label` widget displays a small amount of text. As the name
+implies, most labels are used to label another widget such as a
+`Button`, a `MenuItem`, or a `ComboBox`.
+
+# `Label` as `Buildable`
+
+The `Label` implementation of the `Buildable` interface supports a
+custom `<attributes>` element, which supports any number of `<attribute>`
+elements. The `<attribute>` element has attributes named “name“, “value“,
+“start“ and “end“ and allows you to specify `pango::Attribute` values for
+this label.
+
+An example of a UI definition fragment specifying Pango attributes:
+
+```no_run
+    use gtk::{Button, Label};
+    use gtk::prelude::*;
+
+    // Pressing Alt+H will activate this button
+    let button = Button::new();
+    let label = Label::new_with_mnemonic(Some("_Hello"));
+    button.add(&label);
+```
+
+There’s a convenience function to create buttons with a mnemonic label
+already inside:
+
+
+```no_run
+    use gtk::Button;
+
+    // Pressing Alt+H will activate this button
+    let button = Button::new_with_mnemonic("_Hello");
+```
+
+To create a mnemonic for a widget alongside the label, such as a
+`Entry`, you have to point the label at the entry with
+`Label::set_mnemonic_widget`:
+
+
+```no_run
+    use gtk::{Entry, Label};
+    use gtk::prelude::*;
+
+    // Pressing Alt+H will focus the entry
+    let entry = Entry::new();
+    let label = Label::new_with_mnemonic(Some("_Hello"));
+    label.set_mnemonic_widget(Some(&entry));
+```
+
+# Markup (styled text)
+
+To make it easy to format text in a label (changing colors,
+fonts, etc.), label text can be provided in a simple
+[markup format](https://developer.gnome.org/pygtk/stable/pango-markup-language.html).
+
+Here’s how to create a label with a small font:
+
+```no_run
+    use gtk::Label;
+
+    let label = Label::new(None);
+    label.set_markup("<small>Small text</small>");
+```
+
+(See [complete documentation](https://developer.gnome.org/pygtk/stable/pango-markup-language.html) of available
+tags in the Pango manual.)
+
+The markup passed to `Label::set_markup` must be valid; for example,
+literal <, > and & characters must be escaped as "&lt;", "&gt;", and "&amp;".
```

Comment:
`&lt;` still gets rendered as `<` but in quotes. You probably want `` `&lt;` ``.